large-message-core/src/test/java/com/bakdata/kafka/AmazonS3LargeMessageClientRoundtripTest.java

```diff
@@ -73,6 +74,7 @@ void shouldRoundtrip(final RoundtripArgument argument) {
                 .put(AbstractLargeMessageConfig.BASE_PATH_CONFIG, basePath)
                 .put(AbstractLargeMessageConfig.S3_ENABLE_PATH_STYLE_ACCESS_CONFIG, argument.isPathStyleAccess())
                 .put(AbstractLargeMessageConfig.COMPRESSION_TYPE_CONFIG, argument.getCompressionType())
+                .put(AbstractLargeMessageConfig.S3_SDK_HTTP_CLIENT_BUILDER_CONFIG, DefaultSdkHttpClientBuilder.class.getName())
```

Comment:
Can you think of a way to test behavior? Maybe use a mock?

Reply:
Sth to make sure the custom class is used. Right now the test just continues working but that could also be because the config is not used anywhere. Sth that also covers your use case for customizing the http client